Sources/Orbit/Components/Alert.swift

```diff
@@ -18,15 +30,16 @@ import SwiftUI
 ///
 /// - Note: [Orbit definition](https://orbit.kiwi/components/alert/)
 /// - Important: Component expands horizontally to infinity.
-public struct Alert: View {
+public struct Alert<Content: View>: View {
 
     let title: String
     let description: String
     let icon: Icon.Symbol
-    let buttons: Buttons
+    let buttons: AlertAction
```

Comment:
Shouldn't this be `action: AlertAction` now?

Reply:
Reverted the name to AlertButton.